app/auth/auth_service.ts

```diff
@@ -103,6 +104,10 @@ export class AuthService {
   }
 
   handleLoggedIn(response: user.GetUserResponse) {
+    if (window.opener) {
+      console.log("Posting success message to " + window.location.origin);
```

Comment:
Remove?

Reply:
Initially thought these might be useful for debugging bad behaviors, but they don't really end up telling you much other than whether or not the window was spawned from a popup - which you could check yourself by printing `window.opener` in the console.

Done.